giskard/scanner/report.py

```diff
@@ -296,3 +297,27 @@ def to_wandb(self, run: Optional["wandb.wandb_sdk.wandb_run.Run"] = None) -> Non
             ) from e
 
         run.log({wandb_artifact_name: wandb.Html(html, inject=False)})
+
+    def to_avid(self, filename=None):
+        """Renders the scan report as an AVID report.
+
+        Saves or returns the AVID representation of the scan report.
+
+        Parameters
+        ----------
+        filename : str, optional
+            If provided, the AVID report will be written to the file.
+        """
+        from ..integrations import avid
+
+        reports = [
+            avid.create_report_from_issue(issue=issue, model=self.model, dataset=self.dataset) for issue in self.issues
+        ]
+
+        if filename is not None:
+            with open(filename, "w") as f, warnings.catch_warnings():
+                warnings.filterwarnings("ignore", category=DeprecationWarning)  # we need to support both pydantic 1 & 2
+                f.writelines(r.json() for r in reports)
+            return
+
+        return reports
```

Comment:
This is not working.
```py
with open(filename, "r") as f, warnings.catch_warnings():
    json.load(f)
```
gives
```
JSONDecodeError: Extra data: line 1 column 1130 (char 1129)
```

Reply:
It is not working because the file is not JSON, bug JSONL (each line is a JSON object).

Reply:
Fixed JSONL line break!